packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -230,11 +230,14 @@ export class DeltaManager<TConnectionManager extends IConnectionManager>
 		batch = false,
 		metadata?: any,
 		compression?: string,
+		referenceSequenceNumber?: number,
 	) {
+		// Back-compat ADO:3455
```

Comment:
I don't think we should be adding reference to ADO items.

Reply:
I remember we had a discussion at some point and the consensus was that we shouldn't add links but notes should be fine. I don't think we need to open issues specifically for these things. It's just a TODO with accountability.